Client/Frontend/Login Management/AddCredentialViewController.swift

```diff
@@ -163,6 +181,17 @@ extension AddCredentialViewController: UITableViewDataSource {
     func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
         return 3
     }
+
+    func applyTheme() {
+        let theme = themeManager.currentTheme
+        tableView.separatorColor = theme.colors.borderPrimary
+        tableView.backgroundColor = theme.colors.layer1
+
+        cancelButton.tintColor = theme.colors.actionPrimary
+        saveButton.tintColor = theme.colors.actionPrimary
+
+        updateThemeApplicableSubviews()
```

Comment:
This is now called automatically so you don't need to call it here.

Reply:
Agrh, my next PR has the correct state for this. I will fix it now (in this opened PR) to avoid confusion.